lib/components/InputTags/index.tsx

```diff
@@ -0,0 +1,238 @@
+        display: 'inline-flex',
+    },
+    suggestionsContainer: {
+        display: 'none'
+    },
+    suggestionsContainerOpen: {
+        display: 'block',
+        position: 'absolute',
+        left: 24,
+        marginTop: 40,
+        width: '88%',
+        border: '1px solid #c9c9c9',
+        color: '#292929',
+        backgroundColor: '#FFFFFF',
+        boxShadow: '0 5px 10px 0 rgba(0, 0, 0, 0.12)',
+        fontFamily: 'Helvetica, sans-serif',
+        fontWeight: 500,
+        fontSize: 11,
+        borderRadius: 10,
+        zIndex: 2,
+        overflow: 'hidden',
+    },
+    suggestionsList: {
+        margin: 0,
+        padding: 0,
+        listStyleType: 'none',
+    },
+    suggestion: {
+        display: 'flex',
+        alignItems: 'center',
+        padding: '5px 8px',
+        fontSize: 13,
+        color: '#949494',
+        userSelect: 'none',
+    },
+    suggestionHighlighted: {
+        color: 'white',
+        backgroundColor: '#4f94f8'
+    },
+};
+
+const styles = {
+    '@global': {
+        '.react-tagsinput': {
+            display: 'block',
+            appearance: 'none',
+            border: '1px solid #c9c9c9',
+            padding: 4,
+            boxSizing: 'border-box',
+            borderRadius: 20,
+            minHeight: 34,
+            lineHeight: '34px',
+            fontSize: 11,
+            fontWeight: 500,
+            transition: 'all 250ms ease-out',
+            color: '#292929',
+            backgroundColor: '#FFFFFF',
+        },
+        '.react-tagsinput span': {
+            display: 'flex',
+            flexWrap: 'wrap',
+            alignItems: 'center',
+        },
+        '.react-tagsinput-tag': {
+            display: 'inline-flex',
+            alignItems: 'center',
+            backgroundColor: '#eaeaea',
+            margin: '0 5px 5px 0',
+            padding: 5,
+            fontFamily: 'Helvetica, sans-serif',
+            fontSize: 12,
+            borderRadius: 30,
+            lineHeight: 0,
+        },
+        '.react-tagsinput-tag:last-of-type': {
+            marginBottom: 0,
+        },
+        '.react-tagsinput-remove': {
+            cursor: 'pointer',
+            fontWeight: 'bold',
+        },
+        '.react-tagsinput-tag a::before': {
+            content: " x",
+        },
+        '.react-tagsinput-input': {
+            background: 'transparent',
+            border: 0,
+            color: '#777',
+            fontFamily: 'sans-serif',
+            fontSize: 13,
+            fontWeight: 400,
+            marginBottom: 6,
+            marginTop: 1,
+            outline: 'none',
+            padding: 5,
+            width: 80,
+        },
+    },
+    roundPicture: {
+        margin: '0 8px 0 5px',
+    },
+    suggestionContent: {
+    },
+    renderTag: {
+    },
+    renderTagRoundPicture: {
+        marginRight: 6,
+    },
+    renderTagContent: {
+    },
+    renderTagRemove: {
+    },
+};
+
+interface OwnProps {
+    items: InputTagsItem[],
```

Comment:
What is the difference between items and tags?

Reply:
Items are all the elements to display in the list whereas tags are the selected ones. I removed the tags prop to only have items that have a `selected` boolean